.gitignore

```diff
@@ -27,6 +27,7 @@ _book
 .env.production.local
 secrets.sh
 secrets
+packages/
```

Comment:
all our source code is in this folder LOL

Reply:
:sob: